src/uu/ls/src/ls.rs

```diff
@@ -1111,6 +1111,7 @@ only ignore '.' and '..'.",
                 .long(options::COLOR)
                 .help("Color output based on file type.")
                 .takes_value(true)
+                .possible_values(&["always", "yes", "force", "auto", "tty", "if-tty"])
```

Comment:
`never`, `no` and `none`  are also possible values.

Reply:
You're right.
Fixed with force-pushing.